packages/sqlite_async/lib/src/native/database/connection_pool.dart

```diff
@@ -221,6 +221,15 @@ class SqliteConnectionPool with SqliteQueries implements SqliteConnection {
     // read-only connections first.
     await _writeConnection?.close();
   }
+
+  @override
+  Future<void> refreshSchema() async {
+    final toRefresh = _allReadConnections.toList();
+
+    for (var connection in toRefresh) {
+      await connection.refreshSchema();
```

Comment:
I _think_ the connection-specific locks should work automatically when using these directly, but please confirm that's the case (i.e. that the refresh statement won't be executed in the middle of another transaction).